### PR TITLE
Add Unit Test for transport.flattenConfigRequest

### LINKED ISSUE #318: Add Unit Test for `transport.flattenConfigRequest`
https://github.com/alpstable/gidari/blob/main/internal/transport/transport.go#L231

### PR DESCRIPTION
**Pull request by cecobask**
Resolves #318

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,3 +9,4 @@ bazel-out
 bazel-testlogs
 bazel-driver
 gidari-cli
+.idea
```
Comment (prestonvasquez) [inline .gitignore:12]: @cecobask Can you remove this from the `.gitignore` file?

**Review by prestonvasquez — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (cecobask) [reply .gitignore]: Done 👍🏻
I saw there are VSCode IDE files being ignored and thought it'd help prevent committing IDE files from [GoLand](https://www.jetbrains.com/go/) as well.

Comment (prestonvasquez) [reply .gitignore]: Ah I see, I didn't realize what that was. I should have asked, I will create a follow up ticket to extend the .goignore file. Thank you.

Comment (cecobask) [reply .gitignore]: No problem 🙂 Is there anything else I can do for #318?

**Comment by prestonvasquez** [comment]
Thank you for the contribution @cecobask . Sorry for the delay in reviewing!

**Pull request merged**